joint_limits:
  motor_1:
    has_position_limits: false
    has_velocity_limits: true
    min_velocity: -1500.0
    max_velocity: 1500.0
  motor_2:
    has_position_limits: false
    has_velocity_limits: true
    min_velocity: -1500.0
    max_velocity: 1500.0
  motor_3:
    has_position_limits: false
    has_velocity_limits: true
    min_velocity: -1500.0
    max_velocity: 1500.0
  motor_4:
    has_position_limits: false
    has_velocity_limits: true
    min_velocity: -1500.0
    max_velocity: 1500.0
